code/modules/spells/changeling/split.dm

```diff
@@ -0,0 +1,71 @@
+/spell/changeling/split
+	name = "Split (5)"
+	desc = "We take on the apperance of a species that we have absorbed."
+	abbreviation = "SP"
+	hud_state = "split"
+
+	chemcost = 5
+	required_dna = 1
+	max_genedamage = 0
+	horrorallowed = 0
+	
+/spell/changeling/split/cast(mob/user)
+	if(polling_ghosts)
+		return
+	owner = user.mind
+	polling_ghosts = TRUE
+	visible_message("\You begin to generate a new form.")
+	if(!recruiter)
+		recruiter = new(src)
+		recruiter.display_name = name
+		recruiter.jobban_roles = list("Changeling")
+		recruiter.recruitment_timeout = 30 SECONDS
+	// Role set to Yes or Always
+	recruiter.player_volunteering = new /callback(src, .proc/recruiter_recruiting)
+	// Role set to No or Never
+	recruiter.player_not_volunteering = new /callback(src, .proc/recruiter_not_recruiting)
+
+	recruiter.recruited = new /callback(src, .proc/recruiter_recruited)
+	recruiter.request_player()
+
+	//domutcheck(user, null)  //don't know what this does
```

Comment:
Updates mutations so that the powers and disabilities activate if they're above their activation thresholds. I can't see anything about genetics here though so I guess it's safe to comment it out.